benchmark-service/benchmark/benchmark.py

```diff
@@ -214,26 +217,34 @@ async def compare(request, userdata):  # pylint: disable=unused-argument
     return await render_template('benchmark', request, userdata, 'compare.html', context)
 
 
-async def query_github(app):
-    global START_POINT
-    github_client = app['github_client']
-    request_string = f'/repos/hail-is/hail/commits?since={START_POINT}'
+@router.post('/api/v1alpha/benchmark/create_benchmark')
+async def submit(request):  # pylint: disable=unused-argument
+    app = request.app
+    batch_client = app['batch_client']
+    body = await request.json()
+    commit = body['commit']
```

Comment:
I don't think this should take the full commit. Rather it should take the sha and then we should look up the full status with the gh_client.